frontend/packages/citizen-frontend/src/applications/ChildApplicationsBlock.tsx

```diff
@@ -0,0 +1,200 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import React from 'react'
+import { ApplicationStatus, GuardianApplications } from '~applications/types'
+import { useTranslation } from '~localization'
+import { H2, H3, Label } from '@evaka/lib-components/src/typography'
+import { ContentArea } from '@evaka/lib-components/src/layout/Container'
+import { defaultMargins, Gap } from '@evaka/lib-components/src/white-space'
+import styled from 'styled-components'
+import ListGrid from '@evaka/lib-components/src/layout/ListGrid'
+import { DATE_FORMAT_DATE_TIME, formatDate } from '~util'
+import { ApplicationLink } from '~decisions/ApplicationLink'
+import { Status, statusIcon } from '~decisions/shared'
+import RoundIcon from '@evaka/lib-components/src/atoms/RoundIcon'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faArrowRight, faPlusCircle } from '@evaka/lib-icons'
+import colors from '@evaka/lib-components/src/colors'
+
+const MobileFriendlyListGrid = styled(ListGrid)`
+  @media (max-width: 600px) {
+    grid-template-columns: auto;
+    row-gap: ${defaultMargins.xxs};
+
+    *:nth-child(2n) {
+      margin-bottom: ${defaultMargins.s};
+    }
+  }
+`
+
+const LineBreak = styled.div`
+  width: 100%;
+  border: 1px solid #d8d8d8;
+  margin: 40px 0 20px 0;
+`
+
+const TitleContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  flex-flow: column;
+`
+
+const StatusContainer = styled.div``
+
+const ConfirmationContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  & div {
+    margin-top: 7px;
+  }
+`
+
+const Link = styled.div`
+  & a {
+    color: ${colors.blues.primary};
+    text-decoration: none;
+  }
+`
+
+const Icon = styled(FontAwesomeIcon)`
+  height: 1rem !important;
+  width: 1rem !important;
+  margin-right: 10px;
+`
+
+export default React.memo(function ChildApplicationsBlock({
+  childId,
+  childName,
+  applicationSummaries
+}: GuardianApplications) {
+  const t = useTranslation()
+
+  const applicationStatusToIcon = (
+    applicationStatus: ApplicationStatus
+  ): string => {
+    switch (applicationStatus) {
+      case 'ACTIVE':
+        return 'ACCEPTED'
+      case 'WAITING_PLACEMENT':
+      case 'WAITING_DECISION':
+      case 'WAITING_UNIT_CONFIRMATION':
+      case 'WAITING_MAILING':
+        return 'PROCESSING'
+      case 'WAITING_CONFIRMATION':
+        return 'PENDING'
+      case 'CANCELLED':
+        return 'REJECTED'
+      default:
+        return applicationStatus
+    }
+  }
+
+  return (
+    <ContentArea opaque paddingVertical="L" data-qa={`child-${childId}`}>
+      <TitleContainer>
+        <H2 noMargin data-qa={`title-applications-child-name-${childId}`}>
+          {childName}
+        </H2>
+        <Link>
+          <a href={`/citizen/applications/new`}>
```

Comment:
You should use the react router's `Link` component instead of `<a>` or even more specifically our `Link` in `@evaka/lib-components/src/atoms/Link`.

Also, we already have an `AddButton` component that is basically exactly this. You might as well use that.